### PR TITLE
Add support for skim and skim-tmux

### PR DESCRIPTION
**Pull request by blooper05**
`skim` is a command-line fuzzy finder written in Rust
https://github.com/lotabout/skim

### CODE REVIEW

File: anyframe-functions/selectors/anyframe-selector-auto [hunk 1]
```diff
@@ -23,6 +25,10 @@ if [[ $selector_mode != 'peco' ]] &&
     selector_mode='fzf-tmux'
   elif which fzf > /dev/null 2>&1; then
     selector_mode='fzf'
+  elif which sk-tmux > /dev/null 2>&1; then
+    selector_mode='skim-tmux'
+  elif which sk > /dev/null 2>&1; then
+    selector_mode='skim'
   else
     echo "anyframe: not found any of peco, percol, or fzf" 1>&2
```
Comment (mollifier) [inline anyframe-functions/selectors/anyframe-selector-auto]: Would you add 'skim' to error message.

Comment (blooper05) [reply anyframe-functions/selectors/anyframe-selector-auto]: Thank you for reviewing. I've added it!

**Comment by mollifier** [comment]
LGTM

**Comment by blooper05** [comment]
Hi! When can you merge this PR?

**Comment by mollifier** [comment]
Archive repository

**Pull request closed without merging**